study-builder/studies/pancan/emails/parental_welcome.en.html

```diff
@@ -0,0 +1,13 @@
+#set( $text = {
+"subject": "Thank you for submitting your information to Count Me In",
+"salutation": "Dear Parent,",
+"thank_you": "Thank you for registering to join Count Me In. Your participation isn't just important to our work - it drives everything that we do.",
+"goal": "The goal of this initiative is to generate a large dataset that includes a variety of patient-reported, genomic, clinical, and molecular information that can be shared with the biomedical community in order to accelerate discoveries and better therapies for all types of cancers. Cancer is not a single disease, and each patient story, survey response, medical record, and tumor sample holds part of the puzzle. Our goal is to help the research community better understand the landscape of cancer as a whole and accelerate discoveries by making data more readily available.",
+"stay_in_touch": "We will stay in touch with you via email to let you know about any next steps in your participation, as well as general updates from time to time.",
+"contact_1": "If you have questions at any point and would like to talk to a member of our study team, please reach out to us at",
+"contact_2": "or call",
+"closing": "Sincerely,",
+"study_team": "The Count Me In team",
+"footnote": "For additional updates, follow us on Twitter, Facebook, or Instagram."
```

Comment:
where is still no links for social media, so leave it like a simple text for now

Reply:
added links for twitter and facebook, instagram is still missing